### PR TITLE
New package: fiche-0.9.1

### PR DESCRIPTION
**Pull request by ndowens**
Pastebin server that uses NetCat

### CODE REVIEW

File: srcpkgs/fiche/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+short_desc="Command line pastebin server for sharing terminal output"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="MIT"
+homepage="https://termbin.com"
+distfiles="https://github.com/solusipse/fiche/archive/$version.tar.gz"
+checksum=f0cb279a2c2a4c0d1debcf56785fd8731a1427d2524221678cf69c9aaa85e831
+
+CFLAGS="-lpthread"
+
+system_accounts="fiche"
```
Comment (maxice8) [inline srcpkgs/fiche/template]: please prefix with underscore

File: srcpkgs/fiche/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+fiche_homedir="/var/tmp/fiche"
+make_dirs="/var/log/fiche 0750 fiche fiche
+ /var/tmp 1777 root root
+ /var/tmp/fiche 0750 fiche fiche"
+
+do_install() {
+	vbin fiche
+}
+
+post_install() {
```
Comment (maxice8) [inline srcpkgs/fiche/template]: put this in do_install

File: srcpkgs/fiche/template [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+homepage="https://termbin.com"
+distfiles="https://github.com/solusipse/fiche/archive/$version.tar.gz"
+checksum=f0cb279a2c2a4c0d1debcf56785fd8731a1427d2524221678cf69c9aaa85e831
+
+CFLAGS="-lpthread"
+
+system_accounts="fiche"
+fiche_homedir="/var/tmp/fiche"
+make_dirs="/var/log/fiche 0750 fiche fiche
+ /var/tmp 1777 root root
```
Comment (maxice8) [inline srcpkgs/fiche/template]: don't think creating /var/tmp is necessary

Comment (ndowens) [reply srcpkgs/fiche/template]: Done

File: srcpkgs/fiche/template (hunk 1 above)
Comment (maxice8) [reply srcpkgs/fiche/template]: ping

Comment (ndowens) [reply srcpkgs/fiche/template]: Weird thought I fixed it, will do later

**Comment by ndowens** [comment]
Done

File: srcpkgs/fiche/template [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+short_desc="Command line pastebin server for sharing terminal output"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="MIT"
+homepage="https://termbin.com"
+distfiles="https://github.com/solusipse/fiche/archive/$version.tar.gz"
+checksum=f0cb279a2c2a4c0d1debcf56785fd8731a1427d2524221678cf69c9aaa85e831
+
+CFLAGS="-lpthread"
+
+system_account="fiche"
```
Comment (maxice8) [inline srcpkgs/fiche/template]: please use a prefixed system account if possible.

Comment (ndowens) [reply srcpkgs/fiche/template]: Will do, also I relooked at other templates and it is system_accounts=...

Comment (maxice8) [reply srcpkgs/fiche/template]: please use _fiche

**Pull request closed without merging**